src/integration-test/bootstrap/bootstrap.ts

```diff
@@ -70,16 +71,15 @@ async function waitTillHealthy (appURL: string) {
 
 async function createSmokeTestsUserIfDoesntExist (username: string, userGroup: string, password: string): Promise<void | string> {
   try {
-    return await IdamClient.authenticateUser(username, password)
+    const token = await IdamClient.authenticateUser(username, password)
+
+    return `Bearer ${token}`
```

Comment:
That change is not needed. What is more it makes more sense to not return from `createSmokeTestsUserIfDoesntExist` as return value is never consumed.

Reply:
done